### DISCUSSION TITLE
Clean uninstall Atom on Windows

### DISCUSSION THREAD

**Discussion by markstravels** [atom/atom#22495] [Q&A]
sometime in the past I must have installed Atom tried it out and then uninstalled it.  Now I want to try it again, but I can't get rid of the settings and projects from before.  I uninstalled it twice but that is not good enough.  Where are the files kept I need to delete?

**Comment by icecream17** [comment]
from https://windowsreport.com/uninstall-atom-windows/ https://discuss.atom.io/t/how-to-completely-uninstall-atom-from-windows/17338/20 from https://www.google.com/search?q=delete+atom

1.
![Type "uninstall" in the windows cortana search bar](https://user-images.githubusercontent.com/58114641/120000857-df773d80-bf98-11eb-8b3e-e60c99efff5e.png)
2.
![remove atom](https://user-images.githubusercontent.com/58114641/120001057-059cdd80-bf99-11eb-95fd-b3619525b600.png)
3. Delete `C:\Users\<user name>\.atom`
4. Delete `C:\Users\<user name>\AppData\Local\atom`
5. Delete `C:\Users\<user name>\AppData\Roaming\Atom`
